addons/l10n_es_edi_tbai/models/l10n_es_edi_tbai_document.py

```diff
@@ -401,7 +401,7 @@ def _get_recipient_values(self, partner, is_simplified=False):
         if not partner._l10n_es_is_foreign() and partner.vat:
             recipient_values['nif'] = partner.vat[2:] if partner.vat.startswith('ES') else partner.vat
 
-        elif partner.country_id in self.env.ref('base.europe').country_ids:
+        elif partner.country_id and 'EU' in partner.country_id.country_group_codes:
```

Comment:
You don't always have a country, so the value of the field can be False and the `in` operator will fail